pkg/apis/extensions/types.go

```diff
@@ -97,6 +105,8 @@ type HorizontalPodAutoscalerSpec struct {
 	MinReplicas int `json:"minReplicas"`
 	// MaxReplicas is the upper limit for the number of pods that can be set by the autoscaler. It cannot be smaller than MinReplicas.
 	MaxReplicas int `json:"maxReplicas"`
+	// CPU utilization is the target average CPU utilization (represented as a fraction of requested CPU) over all the pods.
+	CPUUtilization *CPUTargetUtilization `json:"cpuUtilization,omitempty"`
 	// Target is the target average consumption of the given resource that the autoscaler will try to maintain by adjusting the desired number of pods.
 	// Currently two types of resources are supported: "cpu" and "memory".
 	Target ResourceConsumption `json:"target"`
```

Comment:
We definitely want to keep most of the changes from #15549, we are going to submit it after this PR.